queries.sql

```diff
@@ -109,5 +109,59 @@ SELECT O.full_name, COUNT(A."name") as owned
 FROM owners O
 INNER JOIN animals A
 ON A.owners_id = O.id
-GROUP BY O.full_name ORDER BY owned DESC;
-
+GROUP BY O.full_name ORDER BY owned DESC
+LiMIT 1;
+
+SELECT A."name", MAX(VI.date) AS date FROM visits VI
+JOIN vets VE
+ON VE.id = VI.vets_id
+JOIN animals A
+ON VI.animals_id = A.id
+WHERE VE."name" = 'William Tatcher'
+GROUP BY A."name" ORDER BY date DESC
+LIMIT 1;
+
+SELECT COUNT(*) AS count FROM visits VI
+JOIN vets VE
+ON VE.id = VI.vets_id
+WHERE VE."name" = 'Stephanie Mendez';
+
+SELECT vets."name" as vets_name, S."name" as vet_specialization
+FROM vets
+FULL JOIN specializations SP
+ON vets.id = SP.vets_id
+FULL JOIN species S 
+ON SP.species_id = S.id;
+
+SELECT A."name", visits.date FROM visits
+JOIN animals A
+ON visits.animals_id = A.id
+WHERE visits.date >= '2020-04-01' AND visits.date <= '2020-08-30'
+ORDER BY visits.date DESC;
+
+SELECT A."name", COUNT(visits.date) as number_visits FROM visits
+JOIN animals A
+ON visits.animals_id = A.id
+GROUP BY A."name" ORDER BY number_visits DESC
+LIMIT 1;
+
+SELECT A."name", visits.date as date FROM visits
+JOIN animals A
+ON visits.animals_id = A.id
+JOIN vets VE
+ON visits.vets_id = VE.id
+WHERE VE."name" = 'Maisy Smith'
+ORDER BY date
+LIMIT 1;
+
+SELECT VE."name", COUNT(*) as not_specialities FROM visits
+JOIN animals A
+ON visits.animals_id = A.id
+JOIN species
+ON A.species_id = species.id
+JOIN vets VE
+ON visits.vets_id = VE.id
+JOIN specializations as S
+ON S.vets_id = visits.vets_id
+WHERE species.id != S.species_id
+GROUP BY VE."name" ORDER BY not_specialities DESC
```

Comment:
- [ ] Kindly add the following queries, I think you miss to add queries for them, and consider adding a screenshot for them.
![image](https://user-images.githubusercontent.com/53402600/190545227-14e9446c-2e62-474a-857e-081db956741f.png)
![image](https://user-images.githubusercontent.com/53402600/190545277-6c4a3ceb-46f1-4acd-a146-792a12665117.png)